test/sample-test.ts

```diff
@@ -245,11 +245,11 @@ describe("Starknet", function () {
     const transaction = await starknet.getTransaction(txHash);
 
     // Get latest block data
-    const latestBlock = await starknet.getBlock();
+    const latestBlock = await starknet.getBlock({});
```

Comment:
Can you change the `getBlock` function so that the empty object doesn't have to be provided if you want to get the latest block?